extensions/indexes/lucene/src/main/java/org/exist/indexing/lucene/LuceneMatch.java

```diff
@@ -54,18 +47,17 @@ public class LuceneMatch extends Match {
         this.facets = facets;
     }
 
-    private LuceneMatch(LuceneMatch copy) {
+    private LuceneMatch(final LuceneMatch copy) {
         super(copy);
         this.score = copy.score;
         this.luceneDocId = copy.luceneDocId;
         this.query = copy.query;
         this.facets = copy.facets;
-        this.fields = copy.fields;
     }
 
     @Override
-    public Match createInstance(int contextId, NodeId nodeId, String matchTerm) {
-        return null;
+    public Match createInstance(final int contextId, final NodeId nodeId, @Nullable final String matchTerm) {
+        throw new UnsupportedOperationException("Not supported from Lucene Full-Text Index");
```

Comment:
is it useful to report the (content of the) parameters?

Reply:
I don't think that would be heplful here.